preprocess/analysis/test_main.py

```diff
@@ -0,0 +1,3 @@
+def test_hello(client):
+    response = client.get("/")
+    assert response.data == b"Hello World!"
```

Comment:
minor nit: new line. 

Also, if `response.data` returns a byte string instead of a normal string as line 3 suggests, would recommend a comment somewhere about that.  See google's style guide for python doc strings: 
https://google.github.io/styleguide/pyguide.html#381-docstrings

Reply:
I removed this test since it's not necessary to test the base url